app/src/main/java/io/apicurio/studio/event/EventSender.java

```diff
@@ -0,0 +1,21 @@
+package io.apicurio.studio.event;
+
+import io.apicurio.studio.spi.storage.StudioStorage;
+import io.apicurio.studio.spi.storage.model.OutboxEvent;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.inject.Inject;
+
+@ApplicationScoped
+public class EventSender {
+
+    @Inject
+    StudioStorage storage;
+
+    public void onExportedEvent(@Observes OutboxEvent event) {
+        OutboxEvent createdEvent = storage.createEvent(event);
+
+        //Since log-based CDC is expected to be used, the events in the table can be immediately removed.
+        storage.deleteEvent(createdEvent.getId());
```

Comment:
Fascinating.  I didn't understand this point when you mentioned it in the call today.

Reply:
It really depends on how you configure the connector, but if it's configured to ignore the delete statements made to the outbox table, then you can do this, using a blasting 0 space in the disk.

Reply:
You can still send events with a "delete" semantic meaning, but that's still an insert to the outbox table.

Reply:
Yeah understood.  It's very interesting. Does this work for all DBs supported by DBZ?

Reply:
Not sure if it works for all, I think so, since it's the transformation that does this (ignoring the delete event). I can check with the DBZ folks.